flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -504,59 +368,145 @@ async function loadFuel(mcdu, updateView) {
     updateView();
 }
 
+const paxStations = {
```

Comment:
I think those could probably be moved to a global file imported by multiple files instead of repeating the same declaration.

Reply:
Created a new file to host the declaration globally